marlowe-dashboard-client/src/Contract/State.purs

```diff
@@ -294,9 +297,9 @@ transactionsToStep { participants } { txInput, state } =
 timeoutToStep :: State -> Slot -> PreviousStep
 timeoutToStep { participants, executionState } slot =
   let
-    currentContractState = executionState ^. _currentState
+    currentMarloweState = executionState ^. _currentMarloweState
```

Comment:
same discussion as above... maybe something in the lines of `currentSemanticState`